rfs_test/TEST_actioninfo_schema.py

```diff
@@ -35,12 +35,48 @@
 #associated Action, and shall be false if the parameter is not required (optional) to perform the
 #associated Action.
 ###################################################################################################
-
-'''
-Step 1: Try performing an action without the parameter.
-Step 2: If Step 1 failed and Required property is False, fail the assertion. 
-Step 3: Else, pass the assertion. 
-'''
+def Assertion_ACTI104(self,log) :
```

Comment:
The steps for this should be the following:

1. Find a Computer System
  - Go to the System collection
  - Pick any Computer System
2. Check if the Computer System has the "ActionInfo" object for its Reset action
  - If it does not, end the test
3. Check if the Action Info object has any Required parameters
  - If it does not, end the test
4. Perform the action without the provided parameter
  - If a response in the 4XX or 5XX range is returned, the test passes; otherwise the test fails

Reply:
FIxed